netlify/src/main/scala/feral/netlify/NetlifyHttp4sHandler.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.netlify
+
+import cats.effect.kernel.Concurrent
+import cats.syntax.all._
+import fs2.Stream
+import org.http4s.Charset
+import org.http4s.Header
+import org.http4s.Headers
+import org.http4s.HttpRoutes
+import org.http4s.Method
+import org.http4s.Request
+import org.http4s.Response
+import org.http4s.Uri
+import feral.lambda.LambdaEnv
+
+object NetlifyHttp4sHandler {
+
+  def apply[F[_]: Concurrent](
+      routes: HttpRoutes[F]
+  )(implicit env: LambdaEnv[F, NetlifyHttpEvent]): F[Option[NetlifyHttpResult]] =
+    for {
+      event <- env.event
+      method <- Method.fromString(event.httpMethod).liftTo[F]
+      uri <- Uri.fromString(event.path).liftTo[F]
```

Comment:
Currently this is the relative path. Is that correct or should it be the full URL?